perl-lib/OESS/share/nddi.sql

```diff
@@ -73,6 +73,7 @@ CREATE TABLE `circuit_edge_interface_membership` (
   `extern_vlan_id` int(10) NOT NULL,
   `inner_tag` int(10) DEFAULT NULL,
   `circuit_edge_id` int(10) NOT NULL AUTO_INCREMENT,
+  `unit_name` int(11) NOT NULL,
```

Comment:
You're using `select unit from circuit_edge_interface_membership` in the `find_available_unit` methods. Column names must be consistent.